internal/experiment/webconnectivityqa/dnsblocking.go

```diff
@@ -79,7 +79,7 @@ func dnsBlockingBOGON() *TestCase {
 			DNSExperimentFailure:  nil,
 			DNSConsistency:        "inconsistent",
 			XStatus:               4256, // StatusExperimentConnect | StatusAnomalyConnect | StatusAnomalyDNS
-			XDNSFlags:             1,    // AnalysisFlagDNSBogon
+			XDNSFlags:             5,    // AnalysisFlagDNSBogon | AnalysisDNSFlagUnexpectedAddrs
```

Comment:
This change happens because bogons are also unexpected addresses. I think this is fine because we may have cases (some of which @hellais observed for Telstra, IIRC) where a bogon is serving us the correct result but we still want to note somewhere that we have seen a bogon even though such a bogon seems WAI.